src/Spryk/Model/Spryk/Builder/Glue/UpdateGlueValidationSpryk.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Spryk\Model\Spryk\Builder\Glue;
+
+use SprykerSdk\Spryk\Exception\SprykException;
+use SprykerSdk\Spryk\Model\Spryk\Builder\AbstractBuilder;
+use SprykerSdk\Spryk\Model\Spryk\Builder\Resolver\Resolved\ResolvedYmlInterface;
+
+class UpdateGlueValidationSpryk extends AbstractBuilder
+{
+    /**
+     * @var string
+     */
+    protected const IS_OPTIONAL = 'Optional';
+
+    /**
+     * @var string
+     */
+    protected const IS_REQUIRED = 'Required';
+
+    /**
+     * @var string
+     */
+    protected const ARGUMENT_RESOURCE = 'resource';
+
+    /**
+     * @var string
+     */
+    protected const ARGUMENT_METHOD = 'method';
+
+    /**
+     * @var string
+     */
+    protected const ARGUMENT_FIELD = 'field';
+
+    /**
+     * @var string
+     */
+    protected const ARGUMENT_IS_REQUIRED = 'isRequired';
+
+    /**
+     * @var string
+     */
+    protected const ARGUMENT_TARGET_FILE = 'targetFile';
+
+    /**
+     * @return bool
+     */
+    protected function shouldBuild(): bool
+    {
+        $targetFile = $this->getTargetFile();
+        $resolvedTargetFile = $this->fileResolver->resolve($targetFile);
+
+        if (!($resolvedTargetFile instanceof ResolvedYmlInterface)) {
+            return true;
+        }
+
+        $validationStructure = $resolvedTargetFile->getDecodedYml();
+
+        return !isset($validationStructure[$this->getResource()][$this->getMethod()][$this->getField()]);
+    }
+
+    /**
+     * @return void
+     */
+    protected function build(): void
+    {
+        $targetFile = $this->getTargetFile();
+        $resource = $this->getResource();
+        $method = $this->getMethod();
+        $field = $this->getField();
+
+        $resolvedTargetFile = $this->getResolvedYmlFile($targetFile);
+
+        $validationStructure = $resolvedTargetFile->getDecodedYml();
+
+        $validationStructure = $this->populateValidationStructure($validationStructure, $resource, $method, $field);
+
+        $this->createTargetFileDirIfNotExists($targetFile);
+
+        $resolvedTargetFile->setDecodedYml($validationStructure);
+
+        $this->log(sprintf('New Glue validation rule [%s]%s:%s has been added to %s', $method, $resource, $field, $targetFile));
+    }
+
+    /**
+     * @param string $targetFile
+     *
+     * @throws \SprykerSdk\Spryk\Exception\SprykException
+     *
+     * @return \SprykerSdk\Spryk\Model\Spryk\Builder\Resolver\Resolved\ResolvedYmlInterface
+     */
+    protected function getResolvedYmlFile(string $targetFile): ResolvedYmlInterface
+    {
+        if (!is_file($targetFile)) {
+            $this->fileResolver->addFile($targetFile, '[]');
```

Comment:
If the target file doesn't exist it means that the preSpryk was not running. Usually, we do not add things that are missing from Spryks. This would be the first case were we do it but I'm fine with it.